client/my-sites/stats/site.jsx

```diff
@@ -507,40 +540,18 @@ class StatsSite extends Component {
 							showSummaryLink
 							className={ classNames(
 								{
+									// Show "Search terms" as 1/3 when it's not Jetpack ("Downloads" visible) + "Videos" is visible
 									'stats__flexible-grid-item--one-third--two-spaces':
-										! this.isModuleHidden( 'authors' ) && ! supportsUTMStats,
-									'stats__flexible-grid-item--half--large':
-										! this.isModuleHidden( 'authors' ) && ! supportsUTMStats,
-									'stats__flexible-grid-item--half':
-										this.isModuleHidden( 'authors' ) && ! supportsUTMStats,
+										! isJetpack && ! this.isModuleHidden( 'videos' ),
 								},
 								{
-									'stats__flexible-grid-item--one-third--two-spaces': supportsUTMStats,
-									'stats__flexible-grid-item--half--large': supportsUTMStats,
+									// 1/2 for all other cases to stack with Devices or empty space
+									'stats__flexible-grid-item--half': this.isModuleHidden( 'videos' ),
 								},
 								'stats__flexible-grid-item--full--medium'
 							) }
 						/>
 
-						{ ! supportsUTMStats && (
```

Comment:
I removed the old order of modules from the time before UTM because it would complicate significantly the module placement and display conditions.